app/api/login.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Login < Grape::API
+  format :json
+
+  desc 'Login Endpoint'
+  namespace :login do
+    desc 'Login with email and password'
+    params do
+      requires :email, type: String, desc: 'email'
+      requires :password, type: String, desc: 'password'
+    end
+    post do
+
+      user = User.find_by_email params[:email]
+      unless user.present? && user.valid_password?(params[:password])
+        error_msg = 'Invalid Credentials'
+        error!({ 'error_msg' => error_msg }, 401)
```

Comment:
This looks nice! Just a mention, as the error message is used only to raise the error and the message it's quite short, I'd suggest having it directly in the raise error, wdyt?